src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.html

```diff
@@ -23,13 +23,19 @@
                    name="username"
                    formControlName="username"
                    autocomplete="off"
-                   autofocus>
+                   autofocus
+                   ngbTooltip="Empty spaces will be trimmed"
```

Comment:
to be more precise. otherwise there might be a confusion that we might trim all the white spaces.
```suggestion
                   ngbTooltip="White spaces at the beginning and end will be trimmed"
```